sitewit/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Python client for the SiteWit API."""
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```

Comment:
That's always been a mystery for me how we change versions. All I found for now is:

```
Semantic Versioning[1] is a formal convention for specifying compatibility using a three-part version number: major version; minor version; and patch. The patch number is incremented for minor changes and bug fixes which do not change the software's application programming interface (API). The minor version is incremented for releases which add new, but backward-compatible, API features, and the major version is incremented for API changes which are not backward-compatible. 
```